src/strategies_and_params/replicastrategy.jl

```diff
@@ -61,46 +62,82 @@ c{i}_Op{k}_c{j} for operator overlaps.
 
 See [`lomc!`](@ref), [`ReplicaStrategy`](@ref) and [`AbstractHamiltonian`](@ref) (for an
 interface for implementing operators).
+
+If `transform` is not `nothing` then this strategy calculates the overlaps with respect to a 
+similarity transformation of the Hamiltonian (See e.g. [`GutzwillerSampling`](@ref).)
+```math
+    G = f H f^{-1}.
+```
+The expectation value of an operator `A` is then
+```math
+    \\langle A \\rangle = \\langle \\psi | A | \\psi \\rangle 
+        = \\frac{\\langle \\phi | f^{-1} A f^{-1} | \\phi \\rangle}{\\frac{\\langle \\phi | f^{-2} | \\phi \\rangle}
+```
+where 
+```math
+    | \\phi \\rangle = f^{-2} | \\psi \\rangle
+``` 
+is the (right) eigenvector of `G`.
+
+In this case, overlaps of `f^{-2}` are reported first as `c{i}_Op1_c{j}`, and then all 
+transformed operators are reported. That is, for a tuple of input operators `(A, B,...)`, 
+overlaps of `(f^{-1} A f^{-1}, f^{-1} B f^{-1},...)` are reported as `(c{i}_Op2_c{j}, c{i}_Op3_c{j}, ...)`.
+
+In either case, the untransformed vector-vector overlap `c{i}_dot_c{j}` 
+can be omitted with the flag `vecnorm=false`.
 """
-struct AllOverlaps{N,M,O<:NTuple{M,AbstractHamiltonian}} <: ReplicaStrategy{N}
+struct AllOverlaps{N,M,O<:NTuple{M,AbstractHamiltonian},B} <: ReplicaStrategy{N}
     operators::O
 end
 
-function AllOverlaps(num_replicas=2, operator=nothing)
+function AllOverlaps(num_replicas=2, operator=nothing, transform=nothing, vecnorm=true)
```

Comment:
I'd prefer to access the new features through keyword arguments. Having four optional position arguments makes remembering the call signature quite tricky. In fact I would suggest the following:
```suggestion
@deprecate AllOverlaps(num_replicas, operator=nothing) = AllOverlaps(;num_replicas, operator)
function AllOverlaps(; num_replicas=2, operator=nothing, transform=nothing, vecnorm=true)
```
This will still permit the old call signature with one or two positional arguments, but also introduce a new call signature where all arguments are optional keyword arguments.